packages/@aws-cdk-testing/framework-integ/test/aws-eks/test/integ.eks-bootstrap-self-managed-addons.ts

```diff
@@ -0,0 +1,25 @@
+import { App, Stack } from 'aws-cdk-lib';
+import * as eks from 'aws-cdk-lib/aws-eks';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { getClusterVersionConfig } from './integ-tests-kubernetes-version';
+
+class EksClusterStack extends Stack {
+  constructor(scope: App, id: string) {
+    super(scope, id);
+
+    const vpc = new ec2.Vpc(this, 'Vpc', { natGateways: 1 });
+    new eks.Cluster(this, 'EksCluster', {
+      vpc,
+      ...getClusterVersionConfig(this, eks.KubernetesVersion.V1_30),
+      bootstrapSelfManagedAddons: true,
```

Comment:
Thanks for the PR @badmintoncryer - I see from the [CFN docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-eks-cluster.html#cfn-eks-cluster-bootstrapselfmanagedaddons) that by default, the networking addons include vpc-cni, coredns, and kube-proxy. Maybe instead the integ test should pass in `bootstrapSelfManagedAddons: false` and assert somehow that the addons were not installed?